bitcoin/phored/wallet.go

```diff
@@ -101,11 +103,16 @@ func (w *BitcoindWallet) BuildArguments(rescan bool) []string {
 		socksPort := defaultSocksPort(w.controlPort)
 		args = append(args, "-listen", "-proxy:127.0.0.1:"+strconv.Itoa(socksPort), "-onlynet=onion")
 	}
+	if w.separateWallet {
+		args = append(args, "-listen=0", "-datadir="+w.repoPath+"/wallet/", "-rpcuser="+connCfg.User, "-rpcpassword="+connCfg.Pass)
+	}
 	return args
 }
 
 func (w *BitcoindWallet) Start() {
-	w.shutdownIfActive()
```

Comment:
This should shut down only if we *aren't* using a separate wallet. If we are using a separate wallet, we don't want to shut down the current one.